Dockerfile.dapper

```diff
@@ -10,6 +10,7 @@ RUN apt-get update && \
         dosfstools \
         gccgo \
         genisoimage \
+	gettext \
         git \
```

Comment:
👋 may want to check your editor settings; looks like all new lines use tabs, but the existing file uses spaces (here and other files